wwwroot/inc/interface-config.php

```diff
@@ -929,6 +940,50 @@ function ${prefix}_showselectbox(e) {
 	);
 }
 
+function getColorSelect($id = 'color', $selected = NULL)
+{
+
+		if($selected)
+			$class = ' class='.getObjectClass('tag', array('id' => $selected, 'color' => $selected));
+		else
+			$class = '';
+
+		$ret = "<select tabindex='1' name='color' id='${id}' onchange='this.className=this.options[this.selectedIndex].className;'${class}>";
+		$ret .= "<option value=''>-</option>";
+
+		$colors = array(
+				'FFFFFF',
+				'C0C0C0',
+				'808080',
+				'000000',
+				'FF0000',
+				'800000',
+				'FF8000',
+				'FFFF00',
+				'808000',
+				'00FF00',
+				'008000',
+				'00FFFF',
+				'008080',
+				'0000FF',
+				'000080',
+				'FF00FF',
+				'800080'
+		);
+
+		if($selected != NULL && !in_array($selected, $colors))
+			$colors[] = $selected;
+
+		foreach($colors as $color)
+		{
+			$class = getObjectClass('tag', array('id' => $color, 'color' => $color));
+			$ret .= "<option class='${class}' value='$color'".($color == $selected ? " selected" : "" ).">#$color</option>";
+		}
+
+		$ret .= "</select>";
```

Comment:
Would it make sense for `getColorSelect()` to reuse `getSelect()` or at least `getSelectOptions()`?